src/ucp/core/ucp_context.c

```diff
@@ -1599,6 +1599,10 @@ static void ucp_fill_resources_reg_md_map_update(ucp_context_h context)
             context->reg_md_map[mem_type] |= context->dmabuf_reg_md_map;
         }
 
+        if (context->reg_md_map[mem_type] == 0) {
+            continue;
```

Comment:
maybe print that memory registration is not supported by any md?

Reply:
@yosefe I've changed the output info if there's no md supports the specified memory type registration. It has all CI test cases.